src/main/java/scene/SceneManager.java

```diff
@@ -66,11 +64,9 @@ public Scene currentScene() {
     public boolean addScene(Scene scene) {
         boolean add = scenePool.add(scene);
         if (add && enabled) {
-            //a newly added scene is probably raw and uninitialized
-            scene.loadSceneResources();
+            // a newly added scene is probably raw and uninitialized
```

Comment:
this is not necessarily true, a scene may already contain gameObjects to be started.

Reply:
Wel... this isn't from my PR. I just removed the loadSceneResources method since it was unnecessary